niceday-broker/index.js

```diff
@@ -73,22 +77,28 @@ function sleep(ms) {
  * Handle the response from rasa, send each message to the Niceday user.
  * We insert a delay in between messages, so the user has some time to read each message.
  * */
-function onRasaResponse() {
+async function onRasaResponse() {
   if (this.readyState === 4 && this.status === 200) {
     const responseJson = JSON.parse(this.responseText);
-    responseJson.forEach(async (message, i) => {
-      if (ENVIRONMENT === 'prod') {
-        await sleep(i * MESSAGE_DELAY);
-      }
+
+    const processMessage = async (message) => {
       const attachment = {
         replyOfId: null,
-        attachmentIds: [],
+        attachmentIds: message.metadata || [],
       };
-      if ('metadata' in message) {
-        attachment.attachmentIds = message.metadata;
-      }
+
       sendMessage(message.text, parseInt(message.recipient_id, 10), attachment);
-    });
+
+      if (ENVIRONMENT === 'prod') {
```

Comment:
So we're not testing this in dev mode currently? Or is all testing happening through prod?

Reply:
The local testing is in dev mode, but the deployed version is in prod mode, to resemble the actual behavior